settings.gradle

```diff
@@ -48,14 +51,21 @@ include ':services:common'
 include ':services:housetables'
 include ':services:jobs'
 include ':services:tables'
-include ':tables-test-fixtures'
+include ':tables-test-fixtures:tables-test-fixtures-iceberg-1.2'
+include ':tables-test-fixtures:tables-test-fixtures-iceberg-1.5'
 
 include ':cluster:configs'
 include ':cluster:storage'
 include ':cluster:metrics'
 
 include ':scripts:java:tools:dummytokens'
 
-project(':integrations:spark:openhouse-spark-runtime').name = 'openhouse-spark-runtime_2.12'
+project(':integrations:spark:spark-3.1:openhouse-spark-runtime').name = 'openhouse-spark-runtime_2.12'
```

Comment:
thanks, I think this is important to maintain. 
Downstream (li-openhouse) should be able to verify if everything works